cardano-testnet/src/Testnet/Babbage.hs

```diff
@@ -174,6 +172,8 @@ testnet testnetOptions H.Conf {..} = do
 
   -- Copy the cost model
 
+  let numPoolNodes = 3 :: Int
```

Comment:
It can be made configurable, but it shouldn't be configurable until it can be tested to confirm it works.  I think this can be a separate PR.  I will add a TODO.